ghcide/src/Development/IDE/Core/Rules.hs

```diff
@@ -799,7 +799,12 @@ ghcSessionDepsDefinition fullModSummary GhcSessionDepsConfig{..} env file = do
 #endif
             session' <- liftIO $ mergeEnvs hsc moduleNode inLoadOrder depSessions
 
-            Just <$> liftIO (newHscEnvEqWithImportPaths (envImportPaths env) session' [])
+            -- Here we avoid a call to to `newHscEnvEqWithImportPaths`, which creates a new
```

Comment:
@michaelpj Does this sufficiently explain what's going on here? Is there anything we should add?

Reply:
lgtm

Reply:
I would say ask yourself:
1. If I was reading this for the first time, would I know what is going on and why?
2. Has this captured the knowledge that I gained in order to make the fix?